TASVideos.ForumEngine/Node.cs

```diff
@@ -136,6 +136,7 @@ private async Task WriteHref(HtmlWriter w, IWriterHelper h, Func<string, string>
 		w.Attribute("href", href);
 		if (Options != "")
```

Comment:
@adelikat should I do

```suggestion
		if (Options != "" && Name != "url")
```

?

Reply:
@vadosnaprimer What about something like https://github.com/TASVideos/tasvideos/tree/transformurltext?

This captures the idea that we need the tooltip whenever transformUrlText is non-trivial.

Reply:
Yep that's better, adelikat approved too. Please PR.